core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -876,7 +881,7 @@ public String toString(@Nullable AbstractBlockChain chain, @Nullable CharSequenc
             s.append(indent).append("   fee  ").append(fee.multiply(1000).divide(size).toFriendlyString()).append("/kB, ")
                     .append(fee.toFriendlyString()).append(" for ").append(size).append(" bytes\n");
         }
-        if (getVersionShort() == 3 && type.isSpecial())
+        if (getVersionShort() == MAX_STANDARD_VERSION && type.isSpecial())
```

Comment:
This should be DIP2_VERSION, since MAX_STANDARD_VERSION may be raised to 4 at a later time.